scripts/changelog-header.sh

```diff
@@ -15,11 +15,14 @@ x.x.x Release notes (yyyy-MM-dd)
 
 ### Compatibility
 * MongoDB Realm Cloud.
-* APIs are backwards compatible with all previous releases of Realm JavaScript in the 10.x.y series.
-* File format: generates Realms with format v20 (reads and upgrades file format v5 or later for non-synced Realm, upgrades file format v10 for synced Realms).
+* Realm Studio v11.0.0
+* APIs are backwards compatible with all previous releases of Realm JavaScript in the 10.5.y series.
+* File format: generates Realms with format v22 (reads and upgrades file format v5 or later for non-synced Realm, upgrades file format v10 for synced Realms).
 
 ### Internal
-* None.
+* <Either mention core version or upgrade>
+* <Using Realm Core vX.Y.Z>
+* <Upgraded Realm Core from vX.Y.Z to vA.B.C>
```

Comment:
```suggestion
* None.
```

Reply:
@fronck Why would you want to remove this from the template?

Reply:
Woops, wrong file.  Good catch 👍